packages/jest-runner/src/index.ts

```diff
@@ -38,12 +42,17 @@ namespace TestRunner {
   export type TestWatcher = JestTestWatcher;
   export type TestRunnerContext = JestTestRunnerContext;
   export type TestRunnerOptions = JestTestRunnerOptions;
+  export type TestFileEvent = JestTestFileEvent;
 }
 
 /* eslint-disable-next-line no-redeclare */
 class TestRunner {
   private _globalConfig: Config.GlobalConfig;
   private _context: JestTestRunnerContext;
+  public eventEmitter: Emittery;
```

Comment:
```suggestion
  public UNSTABLE_eventEmitter: Emittery;
```

just to have others stay away from it